app/views/administrate/application/_flashes.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if flash.any? %>
+  <div class="flashes">
+    <% flash.each do |key, value| -%>
+      <% next unless value.respond_to?(:html_safe) %>
+      <div class="flash flash-<%= key %>"><%= value.html_safe %></div>
+    <% end -%>
+  </div>
+<% end %>
```

Comment:
Overrides the default administrate partial.
Apparently Administrate sets `flash[:timedout]` to true when the user account is timed out, and true doesn't respond to `html_safe`.

Reply:
Ahhh, I saw that error once this morning. Glad it's an easy fix!